README.md

```diff
@@ -50,6 +52,20 @@ Job:new({
 
 A Lua module that implements a bunch of the things from `pathlib` from Python, so that paths are easy to work with.
 
+### plenary.scandir
+
+`plenery.scandir` is fast recursive file operations. It is similar to unix `find` or `fd` in that it can do recursive scan over a given directory, or a set of directories. 
+
+It offers a wide range of opts for limiting the depth, show hidden and more. `plenary.scan_dir` can be ran asynchronously `on_insert(file, typ)` and `on_exit(files)` as well as synchronously
```

Comment:
Oh f. @tami5 `on_insert` and `on_exit` are available async but `on_insert` and only `on_insert` is also available for sync.

Reply:
:D

Reply:
I think its not clear in the new version. Is it?

Reply:
I think omit the function names, they can read tests and scan_dir file for more usage